reconcile/cli.py

```
@@ -20,8 +33,25 @@ def operator_versions(operators, dry_run):
+)
+@click.option(
+    "--sleep-interval",
+    default=60,
+    type=int,
+    help="Sleep interval between polling attempts in seconds (default: 60)",
+)
+def mgmt_cluster_version(version, dry_run, timeout_minutes, sleep_interval) -> None:
+    try:
+        cluster_upgrade_reconcile(version, dry_run, timeout_minutes, sleep_interval)
+    except (ClusterUpgradeError, RuntimeError, TimeoutError) as e:
+        raise click.ClickException(str(e))
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)